src/Build/Evaluation/IntrinsicFunctions.cs

```diff
@@ -350,6 +351,26 @@ internal static string ValueOrDefault(string conditionValue, string defaultValue
             }
         }
 
+        /// <summary>
+        /// Returns the string after converting all bytes to base 64 (alphanumeric characters plus '+' and '/'), ending in one or two '='.
```

Comment:
Is ending in '=' desirable behavior? It seems to be required for the decoder to work properly, but my gut doesn't like that there are 65 possible characters in base 64 encodings even though one is more of an "end of string" character.

Reply:
It's allowed by the [spec](https://www.ietf.org/rfc/rfc4648.txt) - since Base64 strings must have a multiple of 3 characters, if performing the encoding would result in less that than implementations may add `=` to fill the padding requirement. The `=` padding character is also explicitly part of the the 'base64 alphabet' per the spec too. So no worries about it being odd or incorrect here :)

Reply:
Sounds good! I made an encoder/decoder in college at one point, and the spec our professor provided didn't include padding.